scripts/Sinoquet2009.json

```diff
@@ -0,0 +1,59 @@
+{
+    "citation": "Herv\u00e9 Sinoquet, Sylvain Pincebourde, Boris Adam, Nicolas Don\u00e8s, Jessada Phattaralerphong, Didier Combes, St\u00e9phane Ploquin, Krissada Sangsing, Poonpipope Kasemsap, Sornprach Thanisawanyangkura, G\u00e9raldine Groussier-Bout, and J\u00e9r\u00f4me Casas. 2009. 3-D maps of tree canopy geometries at leaf scale. Ecology 90:283",
```

Comment:
`Hervé Sinoquet, Sylvain Pincebourde, Boris Adam, Nicolas Donès, Jessada Phattaralerphong, Didier Combes, Stéphane Ploquin, Krissada Sangsing, Poonpipope Kasemsap, Sornprach Thanisawanyangkura, Géraldine Groussier-Bout, and Jérôme Casas. 2009. 3-D maps of tree canopy geometries at leaf scale. Ecology 90:283`